test/BF16/Integration/lit.local.cfg

```diff
@@ -19,6 +19,6 @@ def has_support(feature):
 
     return True
 
-# AVX512 and SVE should have BF16 support
-if not has_support('avx512') and not has_support('sve'):
+# AVX512 and SVE should have BF16 support, omit tests if GPU runtime is enabled
+if not has_support('avx512') and not has_support('sve') and not config.tpp_gpu:
```

Comment:
I don't think this does anything. Probably you want sth like:
`if (not has_support('avx512') and not has_support('sve')) or (config.tpp_gpu and config.tpp_gpu != ""):`

Although, I'm not sure why we want to disable them here. It could make sense to disable all non-GPU test but only for the GPU CI.

Reply:
I can confirm, the change "as is" makes test counts equal between Nvidia and Intel due to omitting Bf16 tests on CLX.

Reply:
I think `config.tpp_gpu and config.tpp_gpu != ""` is only convoluted since we originally had "gpu=1". Meanwhile, it's fully settled with strings and `if config.tpp_gpu:` should be fine.

Reply:
> I can confirm, the change "as is" makes test counts equal between Nvidia and Intel due to omitting Bf16 tests on CLX.

It force enables the tests when GPU is enabled. But I guess that's fine if that's the intention.

Reply:
There are 14 BF16 tests so at least that many should show up as unsupported.
`not config.tpp_gpu` is false when TPP_GPU has any value.

Reply:
Hmm, for instance https://buildkite.com/intel/tpp-gpu/builds/250#0189302f-1b9a-4462-87a8-351e0fb47a62 shows 7+161 for both "GPU-Nvidia-Vulkan" and "GPU-Intel-Vulkan" (Unsupported+Passed) while previously we had a discrepancy. I can see your point (15 Bf16 tests vs 7 unsupported).

Reply:
I reverted the changes to Lit config. We can do this later/separately.